e2e-test/scripts/prepare_skywalkingE2E.sh

```diff
@@ -0,0 +1 @@
+go install github.com/apache/skywalking-infra-e2e/cmd/e2e@latest
```

Comment:
FYI @kezhenxu94 @fgksgf It is great to see another new use case of our e2e tooling.

@xjlgod It is better you lock for a specific commit ID rather than latest, in case of breaking change with development/time.

Reply:
@xjlgod We can try the test based on the commit id of tag [1.3.0](https://github.com/apache/skywalking-infra-e2e/commit/1485ae03f0ad90496ad7626a5ae4a6a73a1f6296)

Reply:
Ok, I have changed the version to v1.3.0 in github action and sh file.